aptos-move/aptos-vm-types/src/output.rs

```diff
@@ -43,50 +58,37 @@ impl VMOutput {
     pub fn empty_with_status(status: TransactionStatus) -> Self {
         Self {
             change_set: VMChangeSet::empty(),
+            module_write_set: ModuleWriteSet::empty(),
             fee_statement: FeeStatement::zero(),
             status,
             auxiliary_data: TransactionAuxiliaryData::default(),
         }
     }
 
-    pub fn unpack(
-        self,
-    ) -> (
-        VMChangeSet,
-        u64,
-        TransactionStatus,
-        TransactionAuxiliaryData,
-    ) {
-        (
-            self.change_set,
-            self.fee_statement.gas_used(),
-            self.status,
-            self.auxiliary_data,
-        )
+    pub fn aggregator_v1_delta_set(&self) -> &BTreeMap<StateKey, DeltaOp> {
+        self.change_set.aggregator_v1_delta_set()
     }
 
-    pub fn unpack_with_fee_statement(
-        self,
-    ) -> (
-        VMChangeSet,
-        FeeStatement,
-        TransactionStatus,
-        TransactionAuxiliaryData,
-    ) {
-        (
-            self.change_set,
-            self.fee_statement,
-            self.status,
-            self.auxiliary_data,
-        )
+    pub fn aggregator_v1_write_set(&self) -> &BTreeMap<StateKey, WriteOp> {
+        self.change_set.aggregator_v1_write_set()
```

Comment:
Was wondering why are we getting coverage misses on those functions here. Are those calls actually used? Seems that it need to be called.

Reply:
Yes, they are used in executor task traits, to extract writes from outputs. It might be coverage tool issue?